README.rst

```diff
@@ -71,6 +71,19 @@ for a development install::
     source ./env/bin/activate
     pip install -r requirements.txt -e .
 
+install for nixos
+*****************
+for users of `NixOS` we offer a development shell envoirment that can be
+loaded with::
+
+    nix-shell develop.nix
+
+this will setup the required python environment to run piker, make sure to
+run::
+
+    pip install -r requirements.txt -e .
+
+once after loading the shell
 
 install for tinas
```

Comment:
@guilledk you mind also just dropping this tina section from the readme?

Maybe just toss it in a `tinaland/` subdir that we can focus on more later once we get more serious hackers on normie OSs?